src/kmscon_conf.c

```diff
@@ -715,6 +715,7 @@ int kmscon_conf_new(struct conf_ctx **out)
 		CONF_OPTION_STRING(0, "xkb-compose-file", &conf->xkb_compose_file, ""),
 		CONF_OPTION_UINT(0, "xkb-repeat-delay", &conf->xkb_repeat_delay, 250),
 		CONF_OPTION_UINT(0, "xkb-repeat-rate", &conf->xkb_repeat_rate, 50),
+		CONF_OPTION_BOOL(0, "backspace-sends-delete", &conf->backspace_sends_delete, false),
```

Comment:
I would patch this option to `true` in Debian downstream. Would you prefer to make this a meson option or keep the default like this?

Reply:
What about a meson option only? Usual user shouldn't need to change this frequently.

Reply:
That is enough for my packaging purposes. If the need pops up, the configuration option can always be added back in.